Makefile

```diff
@@ -1,24 +1,26 @@
+R := R
+
 .PHONY: all build check document test
 
 all: document build check
 
 build: document
-	R CMD build .
+	$(R) CMD build .
 
 check: build
-	R CMD check injectr*tar.gz
+	$(R) CMD check injectr*tar.gz
 
 clean:
 	-rm -f injectr*tar.gz
 	-rm -fr injectr.Rcheck
 	-rm -fr src/*.{o,so}
 
 document: clean
-	Rscript -e 'devtools::document()'
-	Rscript -e 'rmarkdown::render("README.Rmd")'
+	$(R) --no-echo -e 'devtools::document()'
```

Comment:
Using no-echo here since --slave seems to be deprecated
https://github.com/rstudio/rstudio/issues/5923